### ISSUE TITLE
Add instructions for testing against the Standard Notes front end

### ISSUE THREAD

**Issue by lanesawyer** [lanesawyer/standardnotes-rs#5]
I haven't even attempted to connect to this yet with the actual Standard Notes app. Right now I have a collection of Postman calls that I've been using to check the requests and responses according to the specification.

So this would be:
1. Get the server to function with the SN front end (I'm not hopeful that it'll work immediately lol)
2. Add instructions for connecting a SN front end to the server

**Comment by teymour-aldridge** [comment]
I imagine it would be possible to crib some of the code from the official implementation to do this.

**Comment by lanesawyer** [comment]
Yeah, I'll probably pull down the web version tonight and run it locally to hit my server. I couldn't get the live web version to talk to localhost (looks like a CORS issue) so building and running the front end locally is likely the way to go.
